src/app/tests/suites/TV_MediaPlaybackCluster.yaml

```diff
@@ -23,6 +23,50 @@ tests:
       cluster: "DelayCommands"
       command: "WaitForCommissionee"
 
+    - label: "Read attribute playback state"
+      command: "readAttribute"
+      attribute: "playback state"
+      response:
+          value: 0
+
+    - label: "Read attribute start time"
+      command: "readAttribute"
+      attribute: "start time"
+      response:
+          value: 255
+
+    - label: "Read attribute duration"
+      command: "readAttribute"
+      attribute: "duration"
+      response:
+          value: 0
+
+    # TODO: Enable once attribute struct is supported
+    # - label: "Read attribute position"
```

Comment:
Again, just mark as disabled, no need to comment out....

Reply:
Uncommented them out. Updated the PR.